test/create.test.js

```diff
@@ -112,3 +111,83 @@ test.skip('tribes.create (opts.addPOBox)', (t) => {
     t.end()
   })
 })
+
+test('create reuses an unused group feed (because of an earlier crash or something)', (t) => {
+  const server = Testbot()
+
+  function countGroupFeeds(cb) {
+    pull(
+      server.metafeeds.branchStream({ old: true, live: false }),
+      pull.filter((branch) => branch.length === 4),
+      pull.map((branch) => branch[3]),
+      pull.filter((feed) => feed.recps),
+      pull.collect((err, feeds) => {
+        if (err) return cb(err)
+        server.metafeeds.printTree(server.id, { id: true }, () => {})
+        return cb(null, feeds.length)
+      })
+    )
+  }
+
+  server.metafeeds.findOrCreate((err, root) => {
+    if (err) t.fail(err)
+
+    t.pass('got root')
+
+    countGroupFeeds((err, num) => {
+      if (err) t.fail(err)
+
+      t.equal(num, 0, 'there are no group feeds yet')
+
+      server.tribes2.create(null, (err) => {
+        if (err) t.fail(err)
+
+        countGroupFeeds((err, num) => {
+          if (err) t.fail(err)
+          t.equal(num, 1, 'there is 1 group feed after we created a group')
+
+          const secret = new SecretKey()
+          server.metafeeds.findOrCreate(
+            {
+              purpose: secret.toString(),
+              feedFormat: 'classic',
+              recps: [
+                { key: secret.toBuffer(), scheme: keySchemes.private_group },
+                root.id,
+              ],
+              encryptionFormat: 'box2',
+            },
+            (err) => {
+              if (err) t.fail(err)
+
+              countGroupFeeds((err, num) => {
+                if (err) t.fail(err)
+                t.equal(
+                  num,
+                  2,
+                  'there is 1 used group feed and 1 dummy one, the empty one we created now'
```

Comment:
It seems to me like there are actually 2 empty group feeds? The first group feed wasn't published on, and neither was the second.

Reply:
on the first one there should be a `group/init` msg

Reply:
Right!